Formula/pandoc.rb

```diff
@@ -22,11 +22,6 @@ class Pandoc < Formula
   depends_on "gmp"
```

Comment:
I think this is a consequence of https://github.com/Homebrew/legacy-homebrew/pull/39134. I'm not sure why pandoc wasn't included in https://github.com/Homebrew/legacy-homebrew/pull/39709.